File: src/main/kotlin/a_basic_control_structures/exercise_4.kt
package a_basic_control_structures

// Develop an algorithm that reads two integers and displays the sum between them.
fun sumBetweenIntegers() {
    print("1st value: ")
    val value1 = readln().toInt()

    print("2nd value: ")
    val value2 = readln().toInt()

    val result = value1.plus(value2)

    println("The sum between $value1 and $value2 is $result.")
}

File: Dart/a_basic_control_structure/exercise_4.dart
// Develop an algorithm that reads two integers and displays the sum between them.

import 'dart:io';

void main() {
  stdout.write('1st value: ');
  int firstValue = int.parse(stdin.readLineSync()!);

  stdout.write('2nd value: ');
  int secondValue = int.parse(stdin.readLineSync()!);

  sum(firstValue, secondValue);
}

void sum(int firstValue, int secondValue) {
  int result = firstValue + secondValue;
  print('The sum between $firstValue and $secondValue is $result.');
}
